### DISCUSSION TITLE
how to  catch  the error  message of http request and  reponse, when the request returns  error messages.

### DISCUSSION THREAD

**Discussion by monica0529** [lowdefy/lowdefy#1178] [Questions and Help]
how to  catch  the specific error  message of http request and  http reponse, when the request returns  error messages. 
including  status code(e.g.409), error detail(e.g.upateData [1.1.3 2022-02-05 17:00:00] Already Existed) and  reason(e.g. Input Conflict), for example，follow is an example:
__________________________________
error: Error: Request Error: Request failed with status code 409; Http response "409: Conflict"; Data: {"status":409,"detail":"upateData [1.1.3 2022-02-05 17:00:00] Already Existed","cause":"Input Conflict"}. at Object.<anonymous> (http://cdn.abc.cn:8088/renderer/233.js:1:29403) at Generator.throw (<anonymous>)

**Comment by Gervwyk** [comment]
Hi please see the docs for try catch on events: https://docs.lowdefy.com/events-and-actions
